common/src/main/java/dev/architectury/networking/simple/SimpleNetworkManager.java

```diff
@@ -49,34 +53,62 @@ private SimpleNetworkManager(String namespace) {
     }
     
     /**
-     * Registers a server -&gt; client message.
+     * Convenience method to register a server -&gt; client message with no packet transformers.
```

Comment:
```suggestion
     * Registers a server -&gt; client message with no packet transformers.
```
I don't think we need the description of this being a helper method